conda/fetch.py

```diff
@@ -283,6 +283,17 @@ def fetch_pkg(info, dst_dir=None, session=None):
     path = join(dst_dir, fn)
 
     download(url, path, session=session, md5=info['md5'], urlstxt=True)
+    if info.get('sig'):
+        from conda.signature import verify
+
+        fn2 = fn + '.sig'
+        url = (info['channel'] if info['sig'] == '.' else
+               info['sig'].rstrip('/') + '/') + fn2
+        log.debug("url=%r" % url)
```

Comment:
From the URL included in the output it is already clear that the file to be downloaded is a signature, but I also modified the debug message.